sapi/fpm/fpm/fpm_signals.c

```diff
@@ -276,6 +282,12 @@ int fpm_signals_init_mask(int *signum_array, size_t size) /* {{{ */
 			return -1;
 		}
 	}
+	memcpy(&child_block_sigset, &block_sigset, sizeof(block_sigset));
```

Comment:
I think this is technically undefined (or at least not recommended for portability) as `sigset_t` should be used only with specified operations. I would prefer to set it in the loop above instead - using the same operations as for `block_sigset`.